### PR TITLE
207 css refactor (388)

### LINKED ISSUE #207: Refactor CSS/HTML

### LINKED ISSUE #388: Documentation for site styles
This was done by samm while working on #207 we're pulling it out into a seperate ticket so we have the docs available to us

### PR DESCRIPTION
**Pull request by sjmog**
Closes #388 

Does not fully close #207

### CODE REVIEW

File: source/elements.html.haml [hunk 1]
```diff
@@ -1,312 +1,32 @@
...
+  .centered-row
+    %ul.tabs.element-tabs
+      %li.tab-header-and-content
+        %a.tab-link.is-active{:href => "#"} Little
+        .tab-content= partial :"elements/little"
+      %li.tab-header-and-content
+        %a.tab-link{:href => "#"} Bigger
+        .tab-content= partial :"elements/bigger"
+      %li.tab-header-and-content
+        %a.tab-link{:href => "#"} Biggest
```
Comment (pitchinvasion) [inline source/elements.html.haml]: update to rails 1.9 hash syntax

**Comment by pitchinvasion** [comment]
All looks good, although we need to check we haven't broken stuff before :shipit:

**Comment by sjmog** [comment]
All looks OK in Chrome (desktop)

**Comment by pitchinvasion** [comment]
:shipit: then

**Comment by sjmog** [comment]
Nooo check mobile + Browserstack first

**Comment by spike01** [comment]
Haha visual QA guidelines amirite. We'll also need to check that this doesn't break anything on apply, blog and ronin-prototype

**Comment by sjmog** [comment]
:+1: for iPhone 4 (emulated)

**Comment by sjmog** [comment]
@pitchinvasion how can I run a local copy of apply, blog, and ronin-prototype using the CSS here?

**Comment by spike01** [comment]
That's... a good point. We need to update the website and publish the new bower version, then you can version bump each of those locally and run check before pulling the trigger

**Comment by sjmog** [comment]
Man, that's painful

**Comment by sjmog** [comment]
ish

**Comment by pitchinvasion** [comment]
no we can bump the version on the pr want me to show you how?

**Comment by sjmog** [comment]
Blog is OK :+1:

**Comment by sjmog** [comment]
Ronin website :+1:

**Comment by sjmog** [comment]
Applications :+1:

**Comment by sjmog** [comment]
All the places are updated, now to merge this?

**Pull request merged**